grml-debootstrap

```diff
@@ -1441,11 +1442,14 @@ prepare_vm() {
 }
 # }}}
 
-# make VM image bootable and unmount it {{{
-finalize_vm() {
+# make VM image bootable {{{
+grub_install() {
   if [ -z "${VIRTUAL}" ] ; then
      return 0
   fi
+  if [ ! "$GRUB_INSTALL" = "yes" ]; then
```

Comment:
I'd propose to switch this into (please also notice the ${...} to stay consistent with the style in the function):

```
if [ "${GRUB_INSTALL}" != "yes" ] ; then
```

and also consider mentioning that we're skipping it because of GRUB_INSTALL, like:

```
einfo "Not installing GRUB as requested via \$GRUB_INSTALL=$GRUB_INSTALL"
```